app/package.json

```diff
@@ -3,7 +3,7 @@
   "productName": "GitHub Desktop",
   "bundleID": "com.github.GitHubClient",
   "companyName": "GitHub, Inc.",
-  "version": "1.2.2",
+  "version": "1.2.3-test1",
```

Comment:
Will this version number need to be updated?

Reply:
Yes, this should be updated as part of generating the changelog for the next version.